guides/v3.1.0/controllers/index.md

```diff
@@ -75,27 +86,16 @@ new `isExpanded` property is true.
 {{/if}}
 ```
 
-You can then define what the action does within the `actions` hook
-of the controller, as you would with a component:
-
-```javascript {data-filename=app/controllers/blog-post.js}
-import Controller from '@ember/controller';
-
-export default Controller.extend({
-  isExpanded: false,
-
-  actions: {
-    toggleBody() {
-      this.toggleProperty('isExpanded');
-    }
-  }
-});
-```
+We can see that if the property `isExpanded` is toggled to true, we will show the body property of the model to the user. This `isExpanded` is stored in the controller.
 
 ### Common questions
 
-<div class="common-question">
-  <h4>Should I use controllers in my application? I've heard they're going away!</h4>
+###### Should we use controllers in my application? I've heard they're going away!
   
-  <p>Yes! Controllers are still an integral part of an Ember application architecture, and generated by the framework even if you don't declare a Controller module explicitly.</p>
-</div>
+Yes! Controllers are still an integral part of an Ember application architecture, and generated by the framework even if you don't declare a Controller module explicitly.
+
+###### When should we create a Controller?
```

Comment:
Can the content of this section be rolled into "When and where to use controllers"?

Reply:
Thoughts on leaving this here? I think it might be quoted a bit and wanted it in the same place for people to see

Reply:
I'm fine with either